invite/src/invite_bot.ts

```diff
@@ -51,10 +51,16 @@ export class InviteBot {
    * event that the invite fails to send for some reason. If left as null, the
    * comment will just say "ask someone for help".
    * Example: 'ampproject/wg-infra'
+   *
```

Comment:
Order the comments on the arguments based on the same order of the arguments :)

Reply:
Done.